### PR TITLE
Rg tabbed interface

### PR DESCRIPTION
**Pull request by rayguna**
Photogram 4.2 - tabbed interface.

### CODE REVIEW

**Review by reherr — APPROVED** [review]
everything looks good to me!

File: app/controllers/users_controller.rb
Comment (CarlosCastanedaDev) [inline app/controllers/users_controller.rb:1]: Looks like you are missing the feed, followers and following actions in this controller. Please add these.

Comment (rayguna) [reply app/controllers/users_controller.rb:1]: Ok. I will work on it.

File: app/views/users/show.html.erb
Comment (CarlosCastanedaDev) [inline app/views/users/show.html.erb:1]: You need to add the tabbed interface to your feed, followers, following and liked views

Comment (rayguna) [reply app/views/users/show.html.erb:1]: Hi Carlos. I have added the functionality to feed, followers, and following views. Liked views was already functional.

**Review by CarlosCastanedaDev — APPROVED** [review]
Nice job, Ray!

In your "users" view folder you could also create a partial for all the tab info and use the partial in the feed, followers, following, liked and show view files. 

You can now merge and move on to the next part.

**Comment by rayguna** [comment]
Created a partial called shared/_tab.html.erb for the feed, followers, following, liked and show view tabs as per Carlos' suggestion.

**Comment by hcontr4** [comment]
Changes align with specifications on assignment page.

**Pull request merged**